2011-04-15  Michael Tautschnig  <mt@debian.org>

	* setup-storage/Fstab.pm: Add comment denoting actual device in case of UUID
		or LABEL being used as identifier.
Index: trunk/lib/setup-storage/Fstab.pm
===================================================================
--- trunk.orig/lib/setup-storage/Fstab.pm
+++ trunk/lib/setup-storage/Fstab.pm
@@ -64,6 +64,10 @@
   $fstab_line[-1] = 0 if ($d_ref->{filesystem} eq "swap");
   $fstab_line[-1] = 0 if ($d_ref->{filesystem} eq "tmpfs");
 
+  # add a comment denoting the actual device name in case of UUID or LABEL
+  push @fstab_line, "# device at install: $dev_name"
+    if ($name =~ /^(UUID|LABEL)=/);
+
   # set the ROOT_PARTITION variable, if this is the mountpoint for /
   $FAI::disk_var{ROOT_PARTITION} = $name
     if ($d_ref->{mountpoint} eq "/");
